retrace-server.spec.in

```diff
@@ -87,6 +88,7 @@ mkdir -p %{buildroot}%{_localstatedir}/log/%{name}
 mkdir -p %{buildroot}%{_localstatedir}/spool/%{name}
 mkdir -p %{buildroot}%{_sysconfdir}/%{name}
 mkdir -p %{buildroot}%{_datadir}/%{name}
+mkdir -p %{buildroot}%{_localstatedir}/lib/retrace
```

Comment:
You can use `%{_sharedstatedir}` instead.

`%{_sharedstatedir}/%{name}` = /var/lib/retrace-server

https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/